src/internal/components/pluggableVisualizations/geoChart/PluggableGeoPushpinChart.tsx

```diff
@@ -59,6 +59,10 @@ export class PluggableGeoPushpinChart extends PluggableBaseChart {
         this.callbacks = callbacks;
         this.geoPushpinElement = element;
         this.initializeProperties(visualizationProperties);
+        // set references to null to clear references
+        this.callbacks.pushData({
+            references: this.references || null,
```

Comment:
could we remove `|| null`?

Reply:
`null` is needed for reset the reference in extended references point. see [condition on AD](https://github.com/gooddata/gdc-analytical-designer/blob/develop/app/reducers/properties_reducer.ts#L39) for details